include/gatekeeper_gk.h

```diff
@@ -61,9 +126,30 @@ struct gk_config {
 
 	struct gk_instance *instances;
 	struct net_config  *net;
+	/*
+	 * XXX The LPM table used by the gk instances.
+	 * Here, we assume that all the gk instances are
+	 * on the same numa node, so that only one global
+	 * LPM table is maintained.
+	 */
+	struct gk_lpm      *lpm_tbl;
```

Comment:
Why don't you have `lpm_tbl` as a `struct` instead of a pointer to save a memory lookup when accessing the fields of `lpm_tbl`?